### ISSUE TITLE
Sign-in issues on sourcegraph.com

### ISSUE THREAD

**Issue by nicksnyder** [sourcegraph/sourcegraph#2520]
Users see "Error: Unknown Error" when attempting to sign in via username and password on sourcegraph.com

![image](https://user-images.githubusercontent.com/754768/53668946-4c6cc300-3c2a-11e9-995b-0a5ebd190dfa.png)

**Comment by ryan-blunden** [comment]
Only sourcegraph.com is experiencing issues. Self-hosted Sourcegraph instances are not affected.

**Comment by nicksnyder** [comment]
We discovered that logins started working after we disabled Cloudflare proxying. Unfortunately we were relying on Cloudflare for SSL and our own SSL cert is expired. Geoffrey is working to generate a new SSL cert for sourcegraph.com to end the p0 incident.

After we have sourcegraph.com working again without Cloudflare, we will investigate why Cloudflare proxying broke for us. It is unclear at this point why this suddenly stopped working this morning.
